src/main/java/ch/njol/skript/expressions/ExprLastSpawnedEntity.java

```diff
@@ -157,7 +223,7 @@ public String toString(@Nullable Event event, boolean debug) {
 			default:
 				assert false;
 		}
-		return "the last " + word + " " + type;
+		return "last " + word + " " + type;
```

Comment:
```suggestion
		return "the last " + word + " " + type;
```
I think it's better to be wordy for these

Reply:
TP a long time ago said to remove the usage of `the` because it can be systematically added when needed. Don't want to add double `the`'s at any point in time.